app/docs/enterprise/0.31-x/installation/ubuntu.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How to install Kong Enterprise and PostgreSQL onto Ubuntu
+---
+
+## Install Kong Enterprise
+
+Download the EE binary for Ubuntu (or Debian) from Bintray
+SFTP the file to your Ubuntu server
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install openssl libpcre3 procps perl
+$ sudo dpkg -i kong-enterprise-edition-0.31-1.zesty.all.deb
+```
+NOTE: EE file may differ in last step above. In addition to Zesty, we currently build for Precise, Trusty, and Xenial.
+
+## Install PostgreSQL
+```bash
+$ sudo apt-get install postgresql
+$ sudo -i -u postgres (puts you into new shell)
+```
+
+Create `kong` user
+
+```bash
+$ psql
+> CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong'; 
+> \q
+$ exit
+```
+
+## Finish Kong Enterprise setup steps
+
+```bash
+# Add contents of your license file (copy & paste)
+$ sudo vi /etc/kong/license.json
+$ sudo cp /etc/kong/kong.conf.default /etc/kong/kong.conf
+# Uncomment and add 'kong' to pg_password line
+$ sudo vi /etc/kong/kong.conf
+```
+
+### Run migrations and start kong
+```bash
+$ kong migrations up
+$ sudo /usr/local/bin/kong start
+```
+
+### (Optional) Setup HTTPie to make commands easier
```

Comment:
I think we can drop this Httpie, given now you using curl example?

Reply:
We could, but I like gently pushing people to use a "better curl" and that's HTTPie.  ⚡️

Reply:
For the sake of simplicity, better stick to single and explicit examples (explicit would be `curl`). or the sake of portability (as well as explicitness), all of our examples are given with `curl`. And for the sake of consistency, we should probably avoid giving examples with multiple tools in some places, and examples with a single tool elsewhere. It also makes the examples longer and harder to follow...

Reply:
Fair enough. I mention `vi` instead of other editors for the same reason. Making it more about the action than the tool. Simple & clean is why I like HTTPie, but same logic justifies simplifying the guidance w/ curl.